src/app/features/shared/components/atoms/Grid/Row.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import styled from "styled-components"
+import { breakpoint } from "@datapunt/asc-ui" 
+import layouts from "@datapunt/asc-ui/lib/theme/default/layouts"
+
+import Column from "app/features/shared/components/atoms/Grid/Column"
+
+/** Rows for page layout
+ * optional props:
+ *    - isFullWidth: by default the max-width of content is set to 1430px, overwrite this with 100% by setting the isFullWidth = true
+ *    - marginBottom, marginTop: add vertical margins by use of the theme's spacing (this includes 'px')
+ * Implementation :
+ * <Row isFullWidth marginBottom={themeSpacing(10)} marginTop={themeSpacing(10)} >
+ */
+
+export type TypeProps = {
+  children: React.ReactNode
+  marginBottom?: any
```

Comment:
type is `number`

Reply:
For consistency the margin is set with the use of `themeSpacing` (a util from ASC), which returns an `any`, eg "24px". When I set type to `number` or `string` I get a TS error.

Reply:
Ok, than it's fine. Will check later.